.github/workflows/pr-platform.yml

```diff
@@ -46,11 +46,11 @@ jobs:
             ${{ runner.os }}-${{ matrix.node_version }}-
 
       - run: npm install --cache ~/.npm --no-audit --progress=false
-      
+
       - run: npm test
-      
-      - run: npm run lint -- --quiet
-      
+
+      - run: npm run lint
```

Comment:
to test: Introduce a change to a file which will fail the linting stage and trigger the `pr:platform` workflow